FrontEndApp/v1-react/src/Components/MainApp/HomeService.js

```diff
@@ -1,81 +1,105 @@
 import axios from "axios";
-const baseurl = process.env.NODE_ENV == "production" ? "/api" : "";
+const baseurl = process.env.NODE_ENV === "production" ? "/api" : "";
 const BACKEND_API_URL = process.env.REACT_APP_BACKEND_API_URL || baseurl;
 
 class HomeService {
   async get_hyperparams(token, data) {
     try {
-      const response = await axios.post(`${BACKEND_API_URL}/v1/hyperparams/get/`, data, {
-        headers: {
-          "Content-Type": "application/json",
-          token: `${token}`,
-        },
-      });
+      const response = await axios.post(
+        `${BACKEND_API_URL}/v1/hyperparams/get/`,
+        data,
+        {
+          headers: {
+            "Content-Type": "application/json",
+            token: `${token}`,
+          },
+        }
```

Comment:
I see this code snippet is being used a lot of time in this file with just variation in the url. Can this be converted to a wrapper or something to reduce code redundancy ?

Reply:
> I see this code snippet is being used a lot of time in this file with just variation in the url. Can this be converted to a wrapper or something to reduce code redundancy ?

Yep we can look into this